docs/snippets/common/storybook-vite-builder-aliasing.js.mdx

```diff
@@ -12,8 +12,6 @@ export default {
   async viteFinal(config) {
     // Merge custom configuration into the default config
     return mergeConfig(config, {
-      // Use the same "resolve" configuration as your app
-      resolve: (await import('../vite.config.js')).default.resolve,
       // Add dependencies to pre-optimization
       optimizeDeps: {
         include: ['storybook-dark-mode'],
```

Comment:
We might want to pick a different addon, as far as I can tell this one is unneeded in v7 and is not compatible either.